endpoints-control/src/main/java/com/google/api/control/Client.java

```diff
@@ -287,21 +281,21 @@ private void logStatistics() {
       return;
     }
     if (statistics.totalReports.get() % statsLogFrequency == 0) {
-      log.info(statistics.toString());
+      log.atInfo().log(statistics.toString());
```

Comment:
If I understand [the flogger docs](https://google.github.io/flogger/best_practice#2-use-the-tostring-method-of-the-logged-value) correctly, calling `.toString()` here is unnecessary.

Reply:
That's not correct--the first argument is always a string. But I see your point that we can save the computation by adding a format string, so done.